src/clr-angular/utils/drag-and-drop/providers/global-drag-mode.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Injectable, Renderer2} from "@angular/core";
+
+@Injectable()
+export class ClrGlobalDragMode {
+    constructor(private renderer: Renderer2) {}
+
+    enter(): void {
```

Comment:
Do we want to raise an issue to fix `ScrollingService` too?

Reply:
Sure, I will bring the discussion to the team.